fe/fe-core/src/main/java/com/starrocks/analysis/DropWorkGroupStmt.java

```diff
@@ -3,10 +3,8 @@
 
 import com.starrocks.sql.ast.AstVisitor;
 
-// Drop WorkGroup specified by name
```

Comment:
Why remove the comment?

Reply:
I don't think this comment is meaningful. If I think it needs to be kept, then I will restore it.